test/unit/chruby_reset_test.sh

```diff
@@ -15,10 +15,13 @@ function test_chruby_reset_hash_table()
 		assertEquals "did not clear the path table" \
 			     "" "$(hash)"
 	elif [[ -n "$BASH_VERSION" ]]; then
+		export LANG=en_US.UTF-8
 		chruby_reset
 
 		assertEquals "did not clear the path table" \
 			     "hash: hash table empty" "$(hash)"
+
+		unset LANG
```

Comment:
`LANG` is usually already set, so many store the original `LANG` value and reset it?

Reply:
Done.